Assignment/training2/src/task1/Features/HomeModal.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import './HomeModal.css';
+import Modal from '../Components/Modal/Modal';
+
+function HomeModal1(e){
+    
+    const [open, setOpen] = useState(false);
+
+    function openModal(e){
+        e.preventDefault();
+        setOpen(!open);
+    };
+    
+    return (
+        <div>
+            <h1 style={{textAlign: 'center'}}>Modal</h1>
+            <div className = "btn--open__modal">
+                <button  onClick={openModal}>
+                    Open Modal
+                </button>
+            </div>
+            <Modal open={open} setOpen={setOpen} />
```

Comment:
@lehuy1012 don't see Close button in your children prop of the modal

Reply:
fixed